### PR TITLE
Update PDB Conversion Task to fix random crash

### PR DESCRIPTION
**Pull request by vancem**
There was a COM bug in the PDB writing code used in the task that
converts portable PDBs to windows PDBs (see https://github.com/dotnet/symreader/pull/70)
This picks up a version with this fix in the build tools.

@tmat  @dagood 

I have confirmed that with this update the fix (which is in the SymUnmanagedWriterImpl.CloseSymWriter method), shows up in the Microsoft.DiaSymReader.dll placed in the bin directory.

### CODE REVIEW

**Comment by vancem** [comment]
Note that after we are comfortable with this, we can pull out the msbuild hacks we have that run the PDB conversion in a sub-process.   It should not be needed anymore.

File: dir.props [hunk 1]
```diff
@@ -86,6 +86,9 @@
     <!-- Need to escape double forward slash (%2F) or MSBuild will normalize to one slash on Unix. -->
     <DnuSourceList Include="https:%2F%2Fdotnet.myget.org/F/dotnet-core/api/v3/index.json" />
     <DnuSourceList Include="https:%2F%2Fdotnet.myget.org/F/dotnet-buildtools/api/v3/index.json" />
+    <DnuSourceList Include="https:%2F%2Fdotnet.myget.org/F/symreader/api/v3/index.json" />
+    <DnuSourceList Include="https:%2F%2Fdotnet.myget.org/F/symreader-native/api/v3/index.json" />
+    <DnuSourceList Include="https:%2F%2Fdotnet.myget.org/F/symreader-converter/api/v3/index.json" />
```
Comment (vancem) [inline dir.props]: We were using really old version of symreader that you could get from nuget.org.   To get the newer myget versions you need to add these feeds.

**Comment by vancem** [comment]
Checking in so I can deploy for CoreFX.

File: dir.props (hunk 1 above)
Comment (dagood) [reply dir.props]: I actually mirrored the packages on https://dotnet.myget.org/gallery/dotnet-buildtools to make sure old versions don't get dropped off the symreader feeds and break the BuildTools build. I don't think any versions of these packages are available on NuGet.org.

I've gone ahead and mirrored the new versions, too.

**Pull request merged**